arenadata/readme.md

```diff
@@ -5,36 +5,25 @@
 Change directory to gpdb sources destination. Make sure that directry doesn't contain binary objects from previous builds. Then run:
 for Ubuntu:
 ```bash
-docker build -t gpdb7_u22:latest -f arenadata/Dockerfile.ubuntu .
-```
-for Rocky Linux:
-```bash
-docker build -t gpdb7_regress:latest -f arenadata/Dockerfile .
+docker build -t gpdb8_u22:latest -f arenadata/Dockerfile.ubuntu .
 ```
 
 CI pushes docker images to the internal registry for each branch. We can pull it with usage of:
 
-* branch name as tag (latest for `adb-7.x` branch)
+* branch name as tag (latest for `adb-8.x` branch)
 * commit hash:
   ```bash
-  docker pull hub.adsw.io/library/gpdb7_regress:1353d81
+  docker pull hub.adsw.io/library/gpdb8_regress:1353d81
   ```
 
 ## Full regression tests suite run
 
 We need to execute [../concourse/scripts/ic_gpdb.bash](../concourse/scripts/ic_gpdb.bash) in container to create demo cluster and run different test suites against it:
 for Ubuntu:
 ```bash
- docker run --name gpdb7_opt_on --rm -it -e TEST_OS=ubuntu \
-  -e MAKE_TEST_COMMAND="-k PGOPTIONS='-c optimizer=on' installcheck-world" \
-  --sysctl "kernel.sem=500 1024000 200 4096" gpdb7_u22:latest \
-  /home/gpadmin/gpdb_src/concourse/scripts/ic_gpdb.bash
-```
-for Rocky Linux:
-```bash
- docker run --name gpdb7_opt_on --rm -it -e TEST_OS=centos \
+ docker run --name gpdb8_opt_on --rm -it -e TEST_OS=ubuntu \
   -e MAKE_TEST_COMMAND="-k PGOPTIONS='-c optimizer=on' installcheck-world" \
-  --sysctl "kernel.sem=500 1024000 200 4096" gpdb7_regress:latest \
+  --sysctl "kernel.sem=500 1024000 200 4096" gpdb8_u22:latest \
```

Comment:
`gpdb8_u22` or `gpdb8_regress`?

Reply:
In harbor, the images for 8X are called gpdb8_u22

Reply:
Maybe then we should use this name in all other places?

Reply:
fixed